NASA-PDS/pds-registry-mgr-elastic
# New LDD version fields (#27)

See https://github.com/NASA-PDS/pds-registry-mgr-elastic/issues/27 for more info.

**Test Instructions**
- Delete old registry if exists. Create new registry.
```
registry-manager delete-registry
registry-manager create-registry
```
- Download latest Cassini LDD from https://github.com/pds-data-dictionaries/ldd-cassini/blob/main/build/development/888e3ee0842d927caf24f7d97bb3fced2de58bdc/1.13.0.0/PDS4_CASSINI_1D00_1500.JSON to some folder, for example, `/tmp/schema/`
- Run Registry Manager `load-dd` command to load the LDD.
```
registry-manager load-dd -dd /tmp/schema/PDS4_CASSINI_1D00_1500.JSON -ns cassini
```
- Check that both `im-version` and `ldd-version` fields have values in Elasticsearch `registry-dd` index for new Cassini LDD. Old LDDs will only have `im-version`. (You can open the following URL in a browser)
```
http://localhost:9200/registry-dd/_search?q=class_name:LDD_Info+and+attr_name:cassini
```
